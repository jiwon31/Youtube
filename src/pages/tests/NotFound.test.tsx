import renderer from "react-test-renderer";
import NotFound from "pages/NotFound";

describe("NotFound", () => {
  it("renders", () => {
    const component = renderer.create(<NotFound />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
