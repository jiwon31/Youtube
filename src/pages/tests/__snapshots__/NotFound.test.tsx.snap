// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders 1`] = `
<h2>
  Not FoundðŸ¥²
</h2>
`;
